README.md

```diff
@@ -15,40 +15,79 @@ To run this project locally, follow these steps:
 
 Clone the repository to your local machine. Replace `YOUR_USERNAME` with your GitHub username:
 
-git clone https://github.com/YOUR_USERNAME/Nc-news-project.git
+```bash
+git clone https://github.com/YOUR_USERNAME/your-repository-name.git
+```
```

Comment:
this is a bit to generalised of a line they want to be cloning your repo from your username.